Navigation.css

```diff
@@ -0,0 +1,38 @@
+@import url(components.css);
+.Nav_links_li, .nav_links_a, button{
```

Comment:
suggestion: maintain consistency in class names. Some names are starting with capitals while some with small.

Reply:
Will take Care from the next time, thank you for the suggestions